packages/terra-demographics-banner/src/selectable-demographics-banner/SelectableDemographicsBanner.jsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import classNames from 'classnames/bind';
+import { KEY_SPACE, KEY_RETURN } from 'keycode-js';
+import Button from 'terra-button';
+import IconInfo from 'terra-icon/lib/icon/IconInformation';
+import DemographicsBannerDisplay from '../DemographicsBannerDisplay';
+import BackgroundTile from './_BackgroundTile';
+import SelectableTile from './_SelectableTile';
+import styles from './SelectableDemographicsBanner.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Application content to display in the banner.
+   */
+  applicationContent: PropTypes.node,
+  /**
+   * Age of the person.
+   */
+  age: PropTypes.string,
+  /**
+   * The persons date of birth
+   */
+  dateOfBirth: PropTypes.string,
+  /**
+   * The person's deceased date. Will display the banner as deceased if this value is provided
```

Comment:
Nit: Please add consistent punctuation (period) to these prop descriptions.

Reply:
[done](https://github.com/cerner/terra-core/pull/2856/files#diff-75915ebe8c2d0aabab6be6449bd17b56)